File: Deficient Number/file.c
#include<stdio.h>
#include<stdlib.h>

int deficient(int num){
    int sum=0;
    for(int i=1;i<num;i++){
        if(num%i==0){
            sum+=i;
        }
    }
    return (sum<num)?1:0;
}
int main()
{
    int num;
    scanf("%d",&num);
    int x=deficient(num);
    x==1 ? printf("Yes it's a Deficient Number.") : printf("No it's not a Deficient Number");
}

File: Deficient Number/file.py
def deficient(n):
    sumval=0
    for i in range(1,n):
        if(n%i==0):
            sumval=sumval+i
    return 1 if(sumval<n) else 0 

num=int(input())
x=deficient(num)
print("Yes it's a Deficient Number.") if(x==True) else print("No it's not a Deficient Number")
